pkg/cmd/cli/cmd/newapp.go

```diff
@@ -423,6 +424,10 @@ func setupAppConfig(f *clientcmd.Factory, out io.Writer, c *cobra.Command, args
 	// Only output="" should print descriptions of intermediate steps. Everything
 	// else should print only some specific output (json, yaml, go-template, ...)
 	output := cmdutil.GetFlagString(c, "output")
+
+	if output == "wide" {
+		return fmt.Errorf("output \"wide\" is undefined")
```

Comment:
IOW, look for a solution to "Make new-app and new-build have the same output options", and what to do with `-o wide` will follow.